lib/loaders/pitcher.js

```diff
@@ -74,7 +74,7 @@ module.exports.pitch = function (remainingRequest) {
     const cacheLoader = cacheDirectory && cacheIdentifier
       ? [`cache-loader?${JSON.stringify({
         cacheDirectory,
-        cacheIdentifier: cacheIdentifier + '-vue-loader-template'
+        cacheIdentifier: cacheIdentifier.replace(/!/g, '') + '-vue-loader-template'
```

Comment:
I think it's better to just hash the whole string with `hash-sum`, since other chars that may have special meanings in inline requests can also cause issues.

Reply:
Fixed at 448e5b953a649171ce3e73dcabd0f85dbce93931